src/equations/equations_parabolic.jl

```diff
@@ -13,3 +13,8 @@ abstract type AbstractCompressibleNavierStokesDiffusion{NDIMS, NVARS} <:
               AbstractEquationsParabolic{NDIMS, NVARS} end
 include("compressible_navier_stokes_2d.jl")
 include("compressible_navier_stokes_3d.jl")
+
+# Resistive and visuous MHD
+abstract type AbstractViscoResistiveMhd{NDIMS, NVARS} <:
+              AbstractEquationsParabolic{NDIMS, NVARS} end
+include("visco_resistive_mhd_3d.jl")
```

Comment:
I would add to some kind of TODO list or a student project ideas list that the 2D implementation of the visco-resistive equations could be interesting. What do you think @sloede ?

Reply:
I increased the simulation time to 10, which is rounded from the 8.33 of the paper. Since such a run would take several hours I changed the time span in the test,